uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/main.js

```diff
@@ -10,15 +10,27 @@ define([
     function ($scope, $rootScope, $rootElement, $log, $modal, $q, FileService, config) {
       $log.debug('Controller: MainCtrl');
 
-      $rootScope.modalDocument = function (data, e) {
+      /**
+       * Opens Document Modal based on passed document data
+       *
+       * @param {string} modalMode 'Edit' or 'New'
```

Comment:
Please add a `-` after the name of the param
`modalMode - Mode of the Modal, 'edit' or 'new'`
same for others